releases.go

```diff
@@ -129,8 +129,10 @@ type ReleaseAssets struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/releases/index.html#create-a-release
 type ReleaseAssetLink struct {
-	Name string `url:"name" json:"name"`
-	URL  string `url:"url" json:"url"`
+	Name     string         `url:"name" json:"name"`
+	URL      string         `url:"url" json:"url"`
+	FilePath *string        `url:"filepath,omitempty" json:"filepath,omitempty"`
+	LinkType *LinkTypeValue `url:"link_type,omitempty" json:"link_type,omitempty"`
```

Comment:
I guess we need to add a `ReleaseAssetLinkOptions` struct instead and use that when making calls to keep things consistent.

Reply:
Sorry, I do not get it. Why a different struct? Or are you saying that we should merge `CreateReleaseLinkOptions`, `UpdateReleaseLinkOptions`, and `ReleaseAssetLink` into one struct?

Reply:
All option types have `url` tags and define all fields as pointers. All structs that are used to parse the response don't. So I thing we should add a `ReleaseAssetLinkOptions` struct that is used in other options structs.

Reply:
Sorry, I do not see `ReleaseAssetLinkOptions` in existing codebase.

I do not get why exactly some other change would be needed. I mean, maybe there has to be some refactor, if you think. But this PR simply adds missing fields to otherwise working API call, no?